content/articles/linkdump-2018-05-04.md

```diff
@@ -0,0 +1,33 @@
+---
+title: "Friday Linkdump for 2018-05-04"
+date: "2018-05-04"
+type: post
+secondary_type: linkdump
+author_name:  "DevTeam"
+author_url: https://compiled.ctl.columbia.edu/authors/
+poster: "poster-linkdump-2018-05-04.jpg"
+lede: "Links we looked at this week, from HTML painting to "
+topics: 
+- Slacklinks
+tags: ["linkdump", "css", "dataviz"]
+---
+
+* [Testing Strategies for React and Redux](https://hacks.mozilla.org/2018/04/testing-strategies-for-react-and-redux/)
+* [Pure CSS Francine](https://github.com/cyanharlow/purecss-francine)
+— HTML/CSS drawing in the style of an 18th-century oil painting. Hand-coded entirely in HTML and CSS.
+* [Geography of the Post](http://cameronblevins.org/gotp/)
+— Map/data visualization of the U.S. Post Offices in the Nineteenth-Century West.
+* [Front-End Developer Handbook 2018](https://frontendmasters.com/books/front-end-handbook/2018/)
+— So, this is what a front-end developer does?
+* [Project Things](https://iot.mozilla.org/)
+— all sorts of Things by Mozilla
+* [How HTTPS works](https://howhttps.works/)
+— Let the CertifiCat, ComPugter, BrowserBird and BadCrab explain security things.
+* [Linux, explained with comics](https://twitter.com/b0rk/media)
+* [On The Turing Completeness of PowerPoint](https://www.youtube.com/watch?v=uNjxe8ShM-8)
+* [Data, R, and a 3-D Printer](http://flowingdata.com/2018/05/01/data-r-and-a-3-d-printer/)
+— 3-D printed charts of BA player shot selection.
+* [GitHub Pages Editor](https://github.com/makinacorpus/github-pages-editor)
```

Comment:
@zmustapha I think this was a dead project that sounded promising but hadn't been updated in four years. I'd vote to kill it.